cv_bridge/CMakeLists.txt

```diff
@@ -12,7 +12,13 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   add_compile_options(-Wall -Wextra)
 endif()
 
+option(CV_BRIDGE_DISABLE_PYTHON "Build Python package" OFF)
```

Comment:
```suggestion
option(CV_BRIDGE_DISABLE_PYTHON "Disable building Python bindings" OFF)
```

Reply:
Thanks for pointing out this. Corrected!